src/net/sourceforge/kolmafia/textui/command/SubmitSpadeDataCommand.java

```diff
@@ -29,13 +31,14 @@ public void run(final String cmd, final String parameters) {
       String contents = data[i];
       String recipient = data[i + 1];
       String explanation = data[i + 2];
-      if (InputFieldUtilities.confirm(
-          "Would you like to send the data \""
-              + contents
-              + "\" to "
-              + recipient
-              + "?\nThis information will be used "
-              + explanation)) {
+      if (autoconfirm
+          || InputFieldUtilities.confirm(
+              "Would you like to send the data \""
+                  + contents
+                  + "\" to "
+                  + recipient
+                  + "?\nThis information will be used "
+                  + explanation)) {
         RequestThread.postRequest(new SendMailRequest(recipient, contents));
       }
     }
```

Comment:
The previous behavior was the spade command always cleared data whether it had been sent or not.  Since we are here is that still what we want?

Reply:
I also thought that was weird but it's outside the scope of this change, in my opinion: I just left it as it was.